__tests__/app.test.js

```diff
@@ -52,20 +52,62 @@ describe("/api/topics", () => {
   });
 });
 
+describe("/api/articles", () => {
+  test("GET 200: responds with an array of all articles sorted by date in descending order, with body property removed", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.articles.length).toBe(13);
+        response.body.articles.forEach((article) => {
+          expect(typeof article.article_id).toBe("number");
+          expect(typeof article.votes).toBe("number");
+          expect(typeof article.title).toBe("string");
+          expect(typeof article.topic).toBe("string");
+          expect(typeof article.author).toBe("string");
+          expect(typeof article.body).toBe("undefined");
+          expect(typeof article.created_at).toBe("string");
+          expect(typeof article.article_img_url).toBe("string");
+        });
+        expect(response.body.articles).toBeSortedBy("created_at", {
+          descending: true,
+        });
+      });
+  });
+  test("GET 200: responds with an array of all articles with a comment_count for each article", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((response) => {
+        response.body.articles.forEach((article) => {
+          expect(typeof article.comment_count).toBe("string");
+        });
+      });
+  });
+});
+
 describe("/api/articles/:article_id", () => {
   test("GET 200: responds with correct article with all keys", () => {
     return request(app)
       .get("/api/articles/1")
       .expect(200)
       .then((response) => {
         expect(response.body.article.article_id).toBe(1);
-        expect(typeof response.body.article.title).toBe("string");
-        expect(typeof response.body.article.topic).toBe("string");
-        expect(typeof response.body.article.author).toBe("string");
-        expect(typeof response.body.article.body).toBe("string");
-        expect(typeof response.body.article.created_at).toBe("string");
-        expect(typeof response.body.article.votes).toBe("number");
-        expect(typeof response.body.article.article_img_url).toBe("string");
+        expect(response.body.article.title).toBe(
+          "Living in the shadow of a great man"
+        );
+        expect(response.body.article.topic).toBe("mitch");
+        expect(response.body.article.author).toBe("butter_bridge");
+        expect(response.body.article.body).toBe(
+          "I find this existence challenging"
+        );
+        expect(response.body.article.created_at).toBe(
+          "2020-07-09T20:11:00.000Z"
+        );
+        expect(response.body.article.votes).toBe(100);
+        expect(response.body.article.article_img_url).toBe(
+          "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
+        );
```

Comment:
Now you're asserting more specific values for your single article, you might want to consider using .toMatchObject syntax to make the assertion more condensed and nicer to interpret